### ISSUE TITLE
Module OpenADAS.impact_excitation_pec raises RuntimeError 'Requested PEC rate is not available'

### ISSUE THREAD

**Issue by pedroLoureiroB** [cherab/core#424]
I'm working with the emissivity of some carbon lines, and for that I need data of PECs, which can be downloaded from https://open.adas.ac.uk/. However, even with all the data of the specific carbon line downloaded, when I try the following:
`adas = OpenADAS(permit_extrapolation = True)`
`excit_c5I = adas.impact_excitation_pec(carbon, 5, (11,9))`
The program raises 'RuntimeError: Requested PEC rate (class=excitation, element=C, charge=5, transition=(11, 9)) is not available'.
I really don't know how I can handle it, because the code works with some carbon lines, for example:
`excit_c1 = adas.impact_excitation_pec(carbon, 2, ('2s2 4f1 2F6.5','2s2 3d1 2D4.5'))`,
while it doesn't with some other lines
If someone can help, I would be very grateful.

**Comment by vsnever** [comment]
Hi @pedroLoureiroB,
You need to manually install PEC for C VI lines in the Cherab atomic data store. To do this, run:
```python
from cherab.openadas.install import install_adf15
from cherab.core.atomic import carbon

install_adf15(carbon, 5, 'adf15/pec96#c/pec96#c_bnd#c5.dat', download=True, header_format='hydrogen')
```
Despite the ion is hydrogen-like, the `header_format` for this specific file must be set to 'hydrogen' and not to 'hydrogen-like'.

**Comment by vsnever** [comment]
Fixed in #425.
